scale/settings.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+
+""" Shared settings and constant values across multiple scaling scripts"""
+
+# Deprecated constant context values used by Kubernetes
+# NAMESPACES = ['datahub', 'prob140', 'stat28']
+# CLUSTER = 'prod'
+# KUBECTL_CONTEXT = 'gke_data-8_us-central1-a_prod'
+GCLOUD_INSTANCE_GROUP = 'gke-dev-default-pool-dbd2a02e-grp'
+GCE_ZONE = "us-central1-a"
+
+# Legacy scaling settings
+USERS_PER_NODE = 7
+#POD_THRESHOLD = 0.9
+#BUMP_INCREMENT = 2
+
+# New scaling settings #TODO: discuss with BIDS
+CAPACITY_PER_NODE = USERS_PER_NODE  # FIXME: should be determined dynamically
```

Comment:
The `request` is what the kubernetes scheduler uses to place pods in various nodes - a node must satisfy all requests for a pod to be scheduled there. We specify `request` for all pods - since we want users to be able to use full 100% of their allocated RAM / CPU (which is why we don't have to worry about RAM surges and stuff). So you can sum the `request` mem for all pods, `capacity` for all nodes and use that to calculate % utilization.